### DISCUSSION TITLE
Create Azure DevOps Demo Generator template?

### DISCUSSION THREAD

**Discussion by sturlath** [abpframework/abp#9748] [Ideas] [answered]
It would be very helpful (only for azure DevOps users of course) if there was a best practice custom template that could be used to get new people up and running with abp.io within few minutes using the [Azure DevOps Services Demo Generator](https://docs.microsoft.com/en-us/azure/devops/demo-gen/?toc=%2Fazure%2Fdevops%2Fdemo-gen%2Ftoc.json&bc=%2Fazure%2Fdevops%2Fdemo-gen%2Fbreadcrumb%2Ftoc.json&view=azure-devops)

[Here is a great starting information and video](https://devblogs.microsoft.com/devops/the-azure-devops-demo-generator/) on it.

**Comment by ebicoglu** [comment, marked as answer]
thanks for the feedback. we will evaluate this
